### PR TITLE
ISPN-12021 creating remote caches

### PR DESCRIPTION
**Pull request by oraNod**
https://issues.redhat.com/browse/ISPN-12021

### CODE REVIEW

File: server/runtime/README.adoc [hunk 1]
```diff
@@ -76,7 +76,6 @@ The following is an example server configuration:
 <cache-container default-cache="default" statistics="false">
       <transport cluster="${infinispan.cluster.name}" stack="udp"/>
       <global-state/>
-      <distributed-cache name="default"/>
```
Comment (pruivo) [inline server/runtime/README.adoc:79]: this configuration example is broken in multiple places. but if you remove the "default" cache, I have the following error:
```
09:56:11,281 FATAL (main) [org.infinispan.SERVER] ISPN080028: Infinispan Server failed to start org.infinispan.commons.CacheConfigurationException: ISPN000432: Missing configuration for default cache 'default' declared on container
```

File: documentation/src/main/asciidoc/topics/proc_creating_caches_cli.adoc [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+. Add cache definitions with the [command]`create cache` command.
++
+* Add a cache definition from an XML or JSON file with the [command]`--file` option.
++
+[source,options="nowrap",subs=attributes+]
+----
+[//containers/default]> create cache --file=configuration.xml mycache
+----
++
+[TIP]
```
Comment (pruivo) [inline documentation/src/main/asciidoc/topics/proc_creating_caches_cli.adoc:22]: shouldn't the "tip" be in the bullet below? I find it weird to read about the "--file" option with a tip about a "--template" option,

File: documentation/src/main/asciidoc/topics/proc_cli_adding_entries.adoc [hunk 3]
```diff
@@ -1,35 +1,31 @@
...
+* Use the [command]`put` command from the context of a cache:
 +
 [source,options="nowrap",subs=attributes+]
 ----
 [//containers/default/caches/mycache]> put hello world
 ----
 +
-[TIP]
+* Use the [command]`--cache=` with the [command]`put` command:
 ====
```
Comment (pruivo) [inline documentation/src/main/asciidoc/topics/proc_cli_adding_entries.adoc:22]: I think this line is breaking the HTML formating of this section.

File: documentation/src/main/asciidoc/topics/proc_cli_connecting.adoc [hunk 4]
```diff
@@ -1,25 +1,43 @@
...
+----
++
+* *Microsoft Windows:*
++
+[source,options="nowrap",subs=attributes+]
+----
+include::cmd_examples/cli_win.adoc[]
+----
++
+. Enter your username and password when prompted.
```
Comment (pruivo) [inline documentation/src/main/asciidoc/topics/proc_cli_connecting.adoc:28]: the username and password are prompted after the `connect` command.
```
$ ./server/runtime/target/infinispan-server-11.0.1-SNAPSHOT/bin/cli.sh 
[disconnected]> connect
Username: dev
Password: ***
```

Comment (pruivo) [reply documentation/src/main/asciidoc/topics/proc_cli_connecting.adoc:28]: I would probably merge it in the next bullet:
"Run the [command]`connect` command and enter your username and password when prompted."

Comment (pruivo) [reply documentation/src/main/asciidoc/topics/proc_cli_connecting.adoc:28]: offtopic: what does `[command]` do?

File: documentation/src/main/asciidoc/topics/con_remote_cache.adoc [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+You create caches at run-time by adding definitions based on `org.infinispan`
+templates or {brandname} configuration through the console, the Command Line
+Interface (CLI), the Hot Rod endpoint, or the REST endpoint.
+
+[IMPORTANT]
+====
+When you create caches at run-time, {brandname} Server replicates your cache
+definitions across the cluster.
+
+You should not add cache definitions to `{server_config}/infinispan.xml`
```
Comment (tristantarrant) [inline documentation/src/main/asciidoc/topics/con_remote_cache.adoc:14]: I would not be so drastic (should not). Something like:

Configurations declared directly in `infinispan.xml` are not automatically synchronized across the cluster. If you wish to use this method, ensure that the configuration is propagated to all of the nodes in your cluster using configuration management tools like Ansible or Chef.

File: server/runtime/README.adoc (hunk 1 above)
Comment (oraNod) [reply server/runtime/README.adoc:79]: ah, yeah. cache-container default-cache="default"

Comment (oraNod) [reply server/runtime/README.adoc:79]: let's just make this example community only. I'll add back the default cache config.

File: documentation/src/main/asciidoc/topics/proc_cli_adding_entries.adoc (hunk 3 above)
Comment (oraNod) [reply documentation/src/main/asciidoc/topics/proc_cli_adding_entries.adoc:22]: great catch

File: documentation/src/main/asciidoc/topics/proc_cli_connecting.adoc (hunk 4 above)
Comment (oraNod) [reply documentation/src/main/asciidoc/topics/proc_cli_connecting.adoc:28]: another good comment. `[command]` adds syntax highlighting downstream. I don't have an example to hand but it's in the markup conventions at: https://redhat-documentation.github.io/asciidoc-markup-conventions/

**Pull request merged**